src/tests/integration/AlbumEndpoints.test.ts

```diff
@@ -15,32 +16,27 @@ const testAlbums = [
   { name: "Album 8", year: 1998, band: "Band B" },
   { name: "Album 9", year: 2001, band: "Band C" },
   { name: "Album 10", year: 2008, band: "Band C" },
-
 ];
 
-let highestAlbumId: number;
-
-describe("Album Controller Tests", () => {
-  beforeEach(async () => {
-    let response: any;
-    for (const album of testAlbums) {
-      response = await Album.create(album);
-    }
-    highestAlbumId = response.id;
-  });
-
-  afterEach(async () => {
-    await Album.destroy({ truncate: true });
-  });
-
+describe("Album Endpoints Tests", () => {
   afterAll(async () => {
+    await Album.destroy({ truncate: true });
+    await Song.destroy({ truncate: true });
     await db.connection.close();
   });
 
   describe("GET album/", () => {
-    test("Should return status code of 200 if no filters are set", async () => {
-      const response = await supertest(app).get("/album");
-      expect(response.statusCode).toEqual(200);
+    let highestAlbumId: number;
```

Comment:
This test setup code should be put into a function and called the same way it's done in the Song endpoint test file